### DISCUSSION TITLE
토이프로젝트 공지

### DISCUSSION THREAD

**Discussion by gina0605** [wafflestudio/seminar-2021#619] [Announcements]
안녕하세요! 와플스튜디오 운영팀입니다.

다음 주에 있을 세미나 5를 끝으로 각 세미나 세션은 종료될 예정입니다. 각 세션 세미나장분들의 판단 하에 세미나를 통과하신 분들께서는 세미나 다음 단계인 토이 프로젝트를 진행하게 됩니다.

토이 프로젝트 일정은 다음과 같습니다.
- 11월 20일: 협업 세미나 & 토이 프로젝트, Sprint 1 시작
- 11월 27일: Sprint 1 마감 및 공유회의 & Sprint 2 시작
- 12월 18일: Sprint 2 마감 및 공유회의 & Sprint 3 시작
- 12월 25일: Sprint 3 마감 & Sprint 4 시작 (회의 생략)
-  1월  8일: Sprint 4 마감 & 중간 총회
-  1월  8일 ~ 29일: 팀별 자유 스프린트
-  1월 29일: 토이 프로젝트 최종 마감 & 총회 & 승격 인원 결정

토이 프로젝트 내용은 다음과 같습니다.
- 실제 존재하는 서비스 중 하나를 팀별로 선정하여 클론 코딩합니다.
  - 클론할 서비스는 아래 기능이 포함되어야 합니다.
    - 페이지네이션
    - 소셜 로그인
    - 글 작성 & 글에 대한 댓글 작성 가능
    - 유저 계정이 존재하고, 유저의 프로필 사진을 업로드하고 수정할 수 있음
  - 서비스는 팀별로 선정하며, 확정하기 전에 운영팀의 확인을 받아야 합니다.
  - 채점 및 공유 편의를 위해 어느 정도 인지도가 높은 프로젝트여야 합니다.
  - 선정한 서비스의 모든 부분을 클론할 필요는 없으며 위 조건에 부합하는 일부를 클론해주시면 됩니다.
- 팀은 랜덤 배정됩니다.
  - 백엔드 3명과 클라이언트 3명이 한 팀이 됩니다.
    - 상황에 따라 불가피하게 2명 혹은 4명이 될 수 있습니다.
  - 팀마다 PM (Project Manager)이 한 분씩 배정될 예정입니다. PM분들께서는 프로젝트에 큰 영향을 미치지는 않으나 팀이 올바른 방향으로 나아가고 있는지 모니터링하고 방향을 잡아주는 역할을 해 주실 예정입니다.

추가로 질문 있으시면 댓글 달아주시면 감사하겠습니다!

*위 일정에 나와있듯이 11월 20일 토요일 오후 2시에는 협업을 위한 세미나가 이루어집니다.
*또한 토이 프로젝트부터는 톡방이 아닌 slack을 활용하여 소통하게 됩니다. 슬랙 워크스페이스에는 당일 저녁에 초대드릴 예정입니다.

감사합니다 :)

**Comment by shp7724** [comment]
안녕하세요. 루키 박신홍입니다.

토이 프로젝트 관련 질문이 있는데요.
백엔드 3명과 클라이언트 3명이 한 팀이 된다고 하셨는데, 기술 스택은 프론트/백엔드가 통일되나요?
예를 들어 백엔드는 3명이 모두 Spring, 프론트엔드는 3명이 모두 iOS 이런 식으로 팀이 구성되나요?

그리고 이후 모든 Sprint 및 총회는 오후 2시에 이루어지는지도 질문드립니다.

감사합니다.

**Reply by woohm402** [reply]
1. 넵 백엔드끼리 / 클라이언트끼리 는 스택이 통일됩니다.
2. 공유회의는 오후 2시가 맞습니다! 총회는 상황 봐서 결정하게 될 것 같습니다 :)

**Reply by woohm402** [reply]
@shp7724 공유 회의 시간 3시로 변경하겠습니다! 혼란을 드려 죄송합니다 🙏
